### PR TITLE
Make Github actions lint match Makefile and check all issues

### PR DESCRIPTION
**Pull request by thom-at-redhat**
# Description

Make Github actions lint match Makefile options and check all issues

Fixes # (issue)

## Type of change

What is it?

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [ ] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I run `make pre-commit` to check fmt/vet/lint/test-no-fdo

### CODE REVIEW

File: pkg/services/files/utils.go [hunk 1]
```diff
@@ -21,7 +21,6 @@ func sanitizePath(destination string, filePath string) (destpath string, err err
 	return
 }
 
-// objectIsWithinMemoryLimit returns a bool result from compare an object with Cloudwatch Maximum Log Message Size
```
Comment (de1987) [inline pkg/services/files/utils.go:24]: could you revert the deletion of this comment?

**Review by de1987 — CHANGES_REQUESTED** [review]

File: pkg/services/files/utils.go (hunk 1 above)
Comment (thom-at-redhat) [reply pkg/services/files/utils.go]: should be reverted.  Please verify and resolve

**Review by de1987 — APPROVED** [review]

**Review by ldjebran — APPROVED** [review]

**Comment by thom-at-redhat** [comment]
@mergifyio refresh

**Comment by thom-at-redhat** [comment]
@mergifyio refresh

**Pull request merged**